__tests__/collection.test.ts

```diff
@@ -435,3 +435,28 @@ describe('random thisArg tests', () => {
 		}, array);
 	});
 });
+
+describe('ensure() tests', () => {
+	const coll = new Collection();
+	coll.set('a', 1);
+	coll.set('b', 2);
+
+	test('set new value if key does not exist', () => {
+		coll.ensure('c', () => 3);
+		expect(coll.size).toStrictEqual(3);
+		expect(coll.has('c')).toBeTruthy();
```

Comment:
We can test the existence and the value in one go with this test:
```suggestion
		expect(coll.get('c')).toStrictEqual(3);
```